ci-operator/config/openshift/release/openshift-release-master__ci-4.14.yaml

```diff
@@ -422,6 +422,16 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-aws-sdn-crun
+- as: e2e-vsphere-ovn
```

Comment:
Title reference adding micro job, should this be an upgrade?  Do we need the same in 4.15?

Reply:
thanks.  Will make it an upgrade job and also add one for 4.15.